maquetado/scss/tech.scss

```diff
@@ -25,22 +25,22 @@
         z-index: -1;
     }
 
-    .tech-logo{
+    .tech-logo {
         width: 100px;
         margin-bottom: 10px;
     }
 
-    .tech-name{
+    .tech-name {
         font-size: $text-medium;
         margin: 5px;
```

Comment:
I think that you can use `,`  selector to apply CSS rules to many classes. For example: `.tech-name, .tech-release, .tech-info { margin: 5px; }`